bin/btmon.py

```diff
@@ -4713,6 +4798,11 @@ def process_calculated(self, packets):
                       options.mqtt_tls,
                       options.mqtt_map or MQTT_MAP,
                       options.mqtt_upload_period or MQTT_UPLOAD_PERIOD))
+    if options.sumo_out:
+      procs.append(SumoLogicProcessor
+                   (options.sumo_url or OEM_URL,
+                    options.sumo_upload_period or OEM_UPLOAD_PERIOD,
+                    options.sumo_timeout or OEM_TIMEOUT))
```

Comment:
sumo_timeout and sumo_upload_period should be documented at the top of the file.  And again, default shouldn't be OEM_*

Reply:
Same